usaspending_api/etl/management/commands/es_rapidloader.py

```diff
@@ -103,112 +101,33 @@ def add_arguments(self, parser):
-
-            new_job = DataJob(job_number, index, fiscal_year, filename)
-
-            if Path(filename).exists():
-                Path(filename).unlink()
-            download_queue.put(new_job)
-
-        printf({"msg": "There are {} jobs to process".format(job_number)})
-
-        process_list = []
-        process_list.append(
-            Process(
-                name="Download Proccess",
-                target=download_db_records,
-                args=(download_queue, es_ingest_queue, self.config),
-            )
-        )
-        process_list.append(
-            Process(
-                name="ES Index Process",
-                target=es_data_loader,
-                args=(self.elasticsearch_client, download_queue, es_ingest_queue, self.config),
-            )
-        )
-
-        process_list[0].start()  # Start Download process
 
-        if self.config["process_deletes"]:
-            process_list.append(
-                Process(
-                    name="S3 Deleted Records Scrapper Process",
-                    target=deleted_transactions,
-                    args=(self.elasticsearch_client, self.config),
-                )
-            )
-            process_list[-1].start()  # start S3 csv fetch proces
-            while process_list[-1].is_alive():
-                printf({"msg": "Waiting to start ES ingest until S3 deletes are complete"})
-                sleep(7)
-
-        process_list[1].start()  # start ES ingest process
-
-        while True:
-            sleep(10)
-            if process_guarddog(process_list):
-                raise SystemExit("Fatal error: review logs to determine why process died.")
-            elif all([not x.is_alive() for x in process_list]):
-                printf({"msg": "All ETL processes completed execution with no error codes"})
-                break
-
-    def complete_process(self) -> None:
-        if self.config["create_new_index"]:
-            set_final_index_config(self.elasticsearch_client, self.config["index_name"])
-            if self.config["skip_delete_index"]:
-                printf({"msg": "Skipping deletion of old indices"})
-            else:
-                printf({"msg": "Closing old indices and adding aliases"})
-                swap_aliases(self.elasticsearch_client, self.config["index_name"])
-
-        if self.config["snapshot"]:
-            printf({"msg": "Taking snapshot"})
-            take_snapshot(self.elasticsearch_client, self.config["index_name"], settings.ES_REPOSITORY)
-
-        if self.config["is_incremental_load"]:
-            msg = "Storing datetime {} for next incremental load"
-            printf({"msg": msg.format(self.config["processing_start_datetime"])})
-            update_last_load_date("es_transactions", self.config["processing_start_datetime"])
-
-
-def process_cli_parameters(options: dict, es_client) -> None:
+def process_cli_parameters(options: dict, es_client):
```

Comment:
Best to leave  `) -> None:` in place

https://stackoverflow.com/a/48223590
https://www.python.org/dev/peps/pep-0484/#the-meaning-of-annotations